[DOC] Edit supportbundle content initial (#22)
Mirantis/mke-docs: content/docs/reference/supportbundle/support-bundle.md

# Support bundle

>For the Alpha.1 release, MKE 4 deploys k0s v1.29.3, which does not inherently
support the [Replicated support bundle
tool](https://troubleshoot.sh/docs/support-bundle/introduction/).

> ## To manually collect a support bundle:

Reviewer: ```suggestion
## Manually collect a support bundle
```

It's a header, it should not have a colon at the end

Author: My intent was for it to present like the pseudo-headers (my term) we use in the other docs sets.

1. SSH into the manager node.
2. Download and install the `support-bundle` tool:

   ```shell
   curl -L https://github.com/replicatedhq/troubleshoot/releases/latest/download/support-bundle_linux_amd64.tar.gz | tar xzvf -
   ```
3. Create a YAML file that details the support bundle configuration.

   Example `support-bundle-worker.yaml` file:

   ```yaml
     apiVersion: troubleshoot.sh/v1beta2
     kind: SupportBundle
     metadata:
       name: sample
     spec:
       collectors:
         - logs:
             selector:
               - app.kubernetes.io/name=blueprint-webhook
           namespace: blueprint-system
           name: logs/blueprint-system
         - logs:
             selector:
               - control-plane=controller-manager
             namespace: blueprint-system
             name: logs/blueprint-system
     ```

     This configuration accomplishes the following:

>      - Capture of cluster information
>      - Setting of cluster resources
>      - Collection of logs from the `blueprint-controller-manager` and
>        `blueprint-operator-webhook` pods, in the `logs/` directory of the
>        output.

Reviewer: ```suggestion
     - Captures cluster information
     - Sets cluster resources
     - Collects logs from the `blueprint-controller-manager` and
       `blueprint-operator-webhook` pods, in the `logs/` directory of the
       output.
```

> ## To collect host information using the k0s-provided YAML file

Reviewer: ```suggestion
## Collect host information using the k0s-provided YAML file
```

Author: My intent was for it to present like the pseudo-headers (my term) we use in the other docs sets.

> 1. Obtain the [k0s-provided YAML file](https://docs.k0sproject.io/stable/support-bundle-worker.yaml).

Reviewer 2: Do we want to again call out here that k0s does not support using this tool for v1.29? Official support was only added for 1.30

Author: Considering the existing note isat the top of the topic and the topic is not very long, it seems like overkill to me to mention it again here.
